packages/react-ui-core/src/Highlighter/__tests__/Highlighter-test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import Highlighter from '../Highlighter'
+
+describe('Highlighter', () => {
+  it('surounds pattern text with span and classname', () => {
+    const wrapper = mount(
+      <Highlighter pattern="test">
+        This is a test sentence for a test
+      </Highlighter>
+    )
+
+    wrapper.setProps({ pattern: 'test' })
+    expect(wrapper.find('[data-tid="highlighter-match"]')).toHaveLength(2)
```

Comment:
Let's set a const to this value `wrapper.find('[data-tid="highlighter-match"]')` instead of looking it up multiple times.  It also makes it easier to change if for some reason the data-tid changes.

Reply:
Sounds good 👍